packages/medusa/src/api/routes/admin/draft-orders/create-line-item.ts

```diff
@@ -92,7 +98,11 @@ export default async (req, res) => {
 
       await cartService
         .withTransaction(manager)
-        .addLineItem(draftOrder.cart_id, line)
+        .addLineItem(
+          draftOrder.cart_id,
+          line,
+          validated.validate_sales_channels
```

Comment:
question: I think we said that for the admin route, we don't have to check the sales channels since an operator can place the product he wants no matter the origin of it

Reply:
yes makes sense, I was unsure about this one but this wasn't specified in the ticket